config/locales/vn.yml

```
@@ -0,0 +1,6 @@
+vn:
```

```suggestion
vi:
```
để là `vi` chứ ko phải `vn` em nhé